src/components/new-request/stats.vue

```diff
@@ -56,7 +56,7 @@
 </template>
 
 <script lang="ts">
-import { Vue, Component, Watch } from 'vue-property-decorator'
```

Comment:
`Watch` wasn't used in this component so I deleted its import.

(I happened upon this file while verifying other things.)

Reply:
awesome , thanks bud